src/Nethermind/Nethermind.JsonRpc/Data/AccessListItemForRpc.cs

```diff
@@ -17,33 +16,36 @@ public class AccessListItemForRpc
         public AccessListItemForRpc(Address address, IReadOnlyCollection<UInt256>? storageKeys)
         {
             Address = address;
-            StorageKeys = storageKeys?.ToArray() ?? Array.Empty<UInt256>();
+            StorageKeys = storageKeys?.ToList() ?? new List<UInt256>();
         }
 
         public Address Address { get; set; }
 
         [JsonProperty(ItemConverterType = typeof(StorageCellIndexConverter))]
-        public UInt256[]? StorageKeys { get; set; }
+        public List<UInt256>? StorageKeys { get; set; }
```

Comment:
Why it was changed to `List` instead of array?

Reply:
I think I needed that to implement `FromAccessList`, but now the implementation of that method is totally different.